website/templates/project/discussions.mako

```diff
@@ -0,0 +1,139 @@
+<%inherit file="project/project_base.mako"/>
+
+<%def name="title()">${node['title']} Discussions</%def>
+
+% if user['can_comment'] or node['has_comments']:
+    <%include file="include/comment_template.mako"/>
+% endif
+
+<div class="col-sm-3">
+    <div class="osf-sidenav hidden-print wiki-sidenav" role="complementary">
+        <ul class="nav bs-sidenav" style="margin: 0;">
+            <!-- Total -->
+            <li ${'class="active"' if comment_target == 'total' else '' | n}>
+                <div class="row">
+                    <a href="${node['url']}discussions/">
+                        <div class="col-xs-12">
+                            <span style="font-size: 16pt;">All Comments</span>
+                        </div>
+                    </a>
+                </div>
+                <hr style="margin: 0"/>
+            </li>
+
+            <!-- Overview -->
+            <li ${'class="active"' if comment_target == 'node' else '' | n}>
+                <div class="row">
+                    <a href="${node['url']}discussions/?page=overview">
+                        <div class="col-xs-12">Project Overview
+                            % if user['unread_comments']['node'] > 0:
+                                <span class="badge pull-right">${user['unread_comments']['node']}</span>
+                            % endif
+                        </div>
+                    </a>
+                </div>
+            </li>
+
+            <!-- wiki -->
+            % if addons:
+                % if 'wiki' in addons_enabled:
+                    <li ${'class="active"' if comment_target == 'wiki' else '' | n}>
+                        <div class="row">
+                            <a href="${node['url']}discussions/?page=wiki">
+                                <div class="col-xs-12">Wiki
+                                    % if user['unread_comments']['wiki'] > 0:
+                                        <span class="badge pull-right">${user['unread_comments']['wiki']}</span>
+                                    % endif
+                                </div>
+                            </a>
+                        </div>
+                    </li>
+                % endif
+            % endif
+
+            <!-- files -->
+            <li ${'class="active"' if comment_target == 'files' else '' | n}>
+                <div class="row">
+                    <a href="${node['url']}discussions/?page=files">
+                        <div class="col-xs-12">Files
+                        % if user['unread_comments']['files'] > 0:
+                            <span class="badge pull-right">${user['unread_comments']['files']}</span>
+                        % endif
+                        </div>
+                    </a>
+                </div>
+            </li>
+        </ul>
+    </div>
+</div>
+
+<div class="col-sm-9">
+    <div class="discussion">
+        % if comment is UNDEFINED:
```

Comment:
Any reason not to use `% if not comment:` here, and elsewhere you're comparing to `UNDEFINED`?